### PR TITLE
Overhaul databases

### PR DESCRIPTION
**Pull request by EdithLansens**

### CODE REVIEW

File: user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Database_Configuration/Configuring_the_database_settings_in_Cube.md [hunk 1]
```diff
@@ -170,68 +127,5 @@ To configure the connection to an [Amazon Keyspaces database](xref:Amazon_Keyspa
    > [!NOTE]
    > In the example screenshots, the cluster health is set to yellow because only one node is used. Single-node clusters are always displayed in yellow.
 
-## Legacy MySQL or SQL database
```
Comment (EdithLansens) [inline user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Database_Configuration/Configuring_the_database_settings_in_Cube.md:173]: This has been moved to the new *Configuring the SQL database settings* page.

File: user-guide/toc.yml [hunk 2]
```diff
@@ -1047,97 +1047,151 @@
...
+                  topicUid: Resources_migration_to_elastic
+            - name: Configuring indexing settings
               topicUid: Configuring_DataMiner_Indexing
-            - name: Configuring the Elasticsearch database
-              topicUid: Configuring_Elasticsearch_Database
+            - name: Maintaining an indexing database
+              topicUid: Configuring_Elasticsearch_indexing_database_backups
+              items:
+                - name: Configuring Elasticsearch indexing database backups
+                  topicUid: Configuring_Elasticsearch_indexing_database_backups
```
Comment (EdithLansens) [inline user-guide/toc.yml:1150]: As of now, the *Configuring Elasticsearch indexing database backups* page is the only subpage under *Maintaining an indexing database*. Perhaps we could limit this to only one main page called *Configuring Elasticsearch indexing database backups* for now and return to this structure once more pages have been added.

**Review by EdithLansens — COMMENTED** [review]
This is a first version of the new Databases section, along with some questions/comments. As a lot of previously existing pages have been moved around, it is important to double-check that they have all been moved to a logical place. Several pages with new information have been added as well, which should be double-checked as well to see whether all added information is correct and whether no important information is missing.

File: user-guide/Advanced_Functionality/Databases/Databases_about.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ uid: Databases_about
```
Comment (EdithLansens) [inline user-guide/Advanced_Functionality/Databases/Databases_about.md:6]: I will restructure/rewrite this *About* page, once we are completely happy with the Databases overhaul and new structure.

File: user-guide/Advanced_Functionality/Databases/Databases_about.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ uid: Databases_about
 
 DataMiner supports multiple different [system data storage architectures](xref:Supported_system_data_storage_architectures).
 
-- Often an **on-premises** [Cassandra database](xref:Cassandra_database) and [Elasticsearch](xref:Elasticsearch_database) or [OpenSearch](xref:OpenSearch_database) database are used.
+- Often an **on-premises** [Cassandra database](xref:Cassandra_database) and [Elasticsearch](xref:Deploying_the_Elasticsearch_database) or [OpenSearch](xref:OpenSearch_database) database are used.
 - Instead of on-premises databases, you can use **managed services from a cloud provider**, e.g. the [Amazon Keyspaces Service](xref:Amazon_Keyspaces_Service), [Amazon OpenSearch Service](xref:Amazon_OpenSearch_Service) or [Azure Managed Instance for Apache Cassandra](xref:Cassandra_database).
 - As a third alternative, soon it will be possible to use **Storage As A Service** on dataminer.services, so you no longer have to maintain the databases yourself, and all the scaling and complexity is taken care of for you.
```
Comment (EdithLansens) [inline user-guide/Advanced_Functionality/Databases/Databases_about.md:11]: Storage As A Service should become the main focus.

**Review by SLC-Laurens-Vercruysse — CHANGES_REQUESTED** [review]
Small change requested on the "multiple elastic" page. 
Since the "multiple elastic" setup merely offloads data to two separate databases without guarantee or syncing mechanism between those two databases, we try to not be promote it that much. For now, we can say something like "this should only be used when dealing with high latency. Low and medium latencies prefer a normal elasticsearch setup"
e.g. for Site redundancy, where the sites are located in different sites but still have reasonably good latency, other possibilities exist. Contact Skyline for more info on those.

**Pull request merged**